File: 프로그래머스/lv1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int half = s.length()/2;
        char[] ch = s.toCharArray();
        answer = ch[half]+"";
        if(s.length()%2 == 0)
            answer = ch[half-1] + answer;
        return answer;
    }
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    if(s.length%2==0)
    var answer = [...s].splice(Math.floor(s.length/2)-1, 2);
    else
    var answer = [...s].splice(Math.floor(s.length/2), 1);
    
    return answer.join('');
}